Makefile

```diff
@@ -80,3 +80,6 @@ test-wechat: ## Run tests for WeChat provider
 
 format: ## Format code
 	@gofmt -s -w pkg
+
+check-format: ## Check if the format looks good.
+	@go fmt ./...
```

Comment:
Specify the `pkg` directory is a better choice, or it will check the Go file format under the `vendor` directory.

Reply:
Should we remove vendor? It is not recommended to keep it in the repo.

Reply:
It ignores vendor by default, I think.

Reply:
I ran the `gofmt -s -w .` yesterday, the `vendor` is included.

> Should we remove vendor? It is not recommended to keep it in the repo.

Yeah, sure.